recipes/perl-log-log4perl/LICENSE

```diff
@@ -0,0 +1,3 @@
+Copyright 2002-2013 by Mike Schilli <m@perlmeister.com> and Kevin Goess <cpan@goess.org>.
```

Comment:
You can delete this file, the license is already being packaged with the source files

Reply:
Makes sense!